examples/tutorials/audio_agent.py

```diff
@@ -0,0 +1,102 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+from datetime import datetime
+
+import numpy as np
+import quaternion  # noqa: F401
+from numpy import ndarray  # noqa: F401
+
+import habitat_sim
+import habitat_sim._ext.habitat_sim_bindings as hsim_bindings
```

Comment:
Importing the bindings directly from examples is not recommended. The relevant bindings should be exposed in a more relevant submodules (like sensors etc...)